### PR TITLE
More (silly) abductee objectives

### PR DESCRIPTION
**Pull request by ghost**
:cl: Reeeeimstupid
add: Silly Abductee objectives. Try not to go crazy trading life stories with Lord Singulo.
/:cl:

Why WOULDN'T you trade life stories w/ lord singulo?

### CODE REVIEW

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 1]
```diff
@@ -456,3 +456,23 @@
...
+
+/datum/objective/abductee/engine
+	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
+	
+/datum/objective/abductee/teamredisbetterthangreen
+	explanation_text = "Tell the AI (or a borg/pAI/drone if there is no AI) some corny technology jokes until it cries for help."
+	
+/datum/objective/abductee/time/New()
+	..()
+	if(istype(ticker.mode, /datum/game_mode/clockwork_cult))
```
Comment (Cheridan) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: >adding specific and blatant mode tells

wew

**Comment by anconfuzedrock** [comment]
I was overly rude for little reason with my criticism before but you still listened, mad respect here, sorry about earlier.

**Comment by gt33** [comment]
Just wanna throw this out here before people say "WHAT IF PEOPLE META THE CLOCK CULT"?   
  
What's the chances of the round being abductor, with an active clock cult, and getting that specific objective?

**Comment by jughu** [comment]
Never understiminate rng
Its like saying what are the chances of the person rolling antag 5 rounds in a row

**Comment by gt33** [comment]
Also, if you do get that objective, chances are if you follow it, you're gonna end up becoming clockcult.

File: code/game/gamemodes/miniantags/abduction/abduction.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Cheridan abductors are pulled from ghosts so they will likely know the round type anyway

Comment (coiax) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: I think this would be funnier if it just spawned regardless of round type.

Comment (gt33) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Good idea. Anyone else think it is?

**Comment by gt33** [comment]
The clock cult objective message shows up no matter the mode now.  Thanks @coiax for the idea

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 2]
```diff
@@ -456,3 +456,19 @@
...
+	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
+	
+/datum/objective/abductee/teamredisbetterthangreen
+	explanation_text = "Tell the AI (or a borg/pAI/drone if there is no AI) some corny technology jokes until it cries for help."
+	
+/datum/objective/abductee/time
+	explanation_text = "Go bug a bronze worshipper to give you a clock."
+		
+/datum/objective/abductee/licky
+	explanation_text = "You must lick anything that you find interesting."
```
Comment (KorPhaeron) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: I can already see the adminhelps from when one player starts emoting that they're licking another

Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: IC issue button

Comment (Iamgoofball) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: IC issue button

**Comment by optimumtact** [comment]
IC issue button

**Pull request merged**